opensrp-chw-hf/src/main/assets/json.form/hiv_outcome.json

```diff
@@ -138,6 +211,15 @@
                 "openmrs_entity_id": "test_not_required_based_on_risk_assessment",
                 "openmrs_entity_parent": ""
               }
+            },
+            {
+              "name": "other",
+              "text": "Other",
+              "meta_data": {
+                "openmrs_entity": "concept",
+                "openmrs_entity_id": "reason_no_action_was_taken",
+                "openmrs_entity_parent": ""
+              }
```

Comment:
This option is not placed in its correct position, the requirement was when others is selected in the question, "Where was the test conducted?" then add a field to capture other places where the test was conducted, but here you have added others in the question "Reasons client was not tested" which is not part of the requirement unless I have missed that requirement, please let me know

Reply:
there are two scenarios where the other option appears the first one Reasons client was not tested" and add an option Others (Specify) and Where was the test conducted? - other option